<launch>
  <arg name="rvizconfig" default="$(find shipbot_ros)/rviz/shipbot.rviz" />
  <node name="shipbot_rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" />
  <rosparam file="$(find shipbot_ros)/config/openmv_thresholds.yaml" />
  <node name="ee_cam_node" pkg="shipbot_ros" type="ee_cam_node" output="screen">
  </node>
</launch>
